tasks/update-component-sbom/update-component-sbom.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: update-component-sbom
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Update component-level SBOM with purls with release-time info.
+  params:
+    - name: dataJsonPath
+      description: Relative path to the JSON data file in the workspace.
+    - name: downloadedSbomPath
+      description: |
+        Path to the directory holding previously downloaded SBOMs to be updated.
+      type: string
+      default: downloaded-sboms
+  workspaces:
+    - name: data
+      description: The workspace where the SBOM files reside.
+  results:
+    - name: sbomPath
+      description: Relative path to the updated component-level SBOM in the data workspace.
+  steps:
+    - name: update-component-sbom-purls
+      image: quay.io/konflux-ci/release-service-utils:221d71a4f6b1a50b36b685aa20d86d7df9de33fc
+      script: |
+        #!/usr/bin/env bash
+        set -eux
+        
+        SBOM_PATH="$(dirname "$(params.dataJsonPath)")/$(params.downloadedSbomPath)"
```

Comment:
This is going to be the sbomPath result from push-rpm-data-to-pyxis, no? So do we really need to find the dirname again? Isn't the param enough? So `INPUT_PATH="$(workspaces.data.path)/$(params.downloadedSbomPath)"`?

Reply:
Yeah I think you're right. Updated.

Since we're relying on subdirs to prevent conflicts from multiple pipelineruns, I also removed the default value in the `downloadedSbomPath` param and made it required instead of optional.